validation/governed-by-optimism_test.go

```diff
@@ -0,0 +1,23 @@
+package validation
+
+import (
+	"testing"
+
+	. "github.com/ethereum-optimism/superchain-registry/superchain"
+	"github.com/ethereum-optimism/superchain-registry/validation/standard"
+
+	// "github.com/ethereum/go-ethereum/common"
+
+	"github.com/stretchr/testify/require"
+)
+
+func testGovernedByOptimism(t *testing.T, chain *ChainConfig) {
+	chainID := chain.ChainID
+	superchain := OPChains[chainID].Superchain
+
+	optimismMultisig := standard.Config.MultisigRoles[superchain].KeyHandover.L1.Universal["ProxyAdmin"]["owner()"]
+
+	if chain.GovernedByOptimism == true {
+		require.Equal(t, chain.Addresses.ProxyAdmin.String(), optimismMultisig, "Chains using Optimism governance must have their ProxyAdmin set to the Optimism multisig")
```

Comment:
1. You should permute the expected and actual slots for `require.Equal`
2. You need to use `ProxyAdminOwner` not `ProxyAdmin`		

```suggestion
		require.Equal(t, optimismMultisig, chain.Addresses.ProxyAdminOwner.String(), "Chains using Optimism governance must have their ProxyAdminOwner set to the Optimism multisig")
```